src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/hmi/rc_on_remote_control_settings_notification.cc

```diff
@@ -77,40 +79,44 @@ void RCOnRemoteControlSettingsNotification::DisallowRCFunctionality() {
 }
 
 void RCOnRemoteControlSettingsNotification::Run() {
-    LOG4CXX_AUTO_TRACE(logger_);
+  LOG4CXX_AUTO_TRACE(logger_);
 
-    if (!(*message_)[app_mngr::strings::msg_params].keyExists(message_params::kAllowed)) {
-      LOG4CXX_DEBUG(logger_,
-                    "Notification is ignored due to \"allow\" parameter absense");
-      LOG4CXX_DEBUG(logger_, "RC Functionality remains unchanged");
-      return;
-    }
+  if (!(*message_)[app_mngr::strings::msg_params].keyExists(
+          message_params::kAllowed)) {
+    LOG4CXX_DEBUG(logger_,
+                  "Notification is ignored due to \"allow\" parameter absense");
+    LOG4CXX_DEBUG(logger_, "RC Functionality remains unchanged");
+    return;
+  }
 
-    const bool is_allowed = (*message_)[app_mngr::strings::msg_params][message_params::kAllowed].asBool();
-    if (is_allowed) {
-      hmi_apis::Common_RCAccessMode::eType access_mode =
-          hmi_apis::Common_RCAccessMode::INVALID_ENUM;
-      LOG4CXX_DEBUG(logger_, "Allowing RC Functionality");
-      if ((*message_)[app_mngr::strings::msg_params].keyExists(message_params::kAccessMode)) {
-        const std::string access_mode_str =
-            (*message_)[app_mngr::strings::msg_params][message_params::kAccessMode].asString();
+  const bool is_allowed =
+      (*message_)[app_mngr::strings::msg_params][message_params::kAllowed]
+          .asBool();
+  if (is_allowed) {
```

Comment:
@BSolonenko is it qt style fix? Please use script for check style

Reply:
@LuxoftAKutsan checked in https://github.com/LuxoftAKutsan/sdl_core/pull/275/commits/5a296e130d1911c156d105e21d0ef00d4c9b97b1